src/prepack-cli.js

```diff
@@ -274,9 +285,16 @@ function run(
             `Invalid debugger diagnostic severity: ${arg}`
           );
           debuggerConfigArgs.diagnosticSeverity = arg;
+          reproArguments.push("--debugDiagnosticSeverity", arg);
           break;
         case "debugBuckRoot":
-          debuggerConfigArgs.buckRoot = args.shift();
+          let buckRoot = args.shift();
+          debuggerConfigArgs.buckRoot = buckRoot;
+          if (debugReproArgs) debugReproArgs.buckRoot = buckRoot;
+          // Use $(pwd)  instead of argument so repro script can be run from
```

Comment:
Per my understanding, `debugBuckRoot` is only used by `SourceMapManager` while buck is being used. In `repro replay` mode, there should never be any buck so this parameter is never needed and should not be added into `reproArguments`

Reply:
The "buck" part of buckRoot does not refer to an invocation of the buck build system -- it refers to the format of the sourcemaps when they're generated by Buck. 

The buckRoot is still necessary because the format of the sourcemaps hasn't changed. If we want to open the correct original files, we still need to perform the same relative/absolute translation which requires the buckRoot.